reproject/interpolation/high_level.py

```diff
@@ -16,7 +16,8 @@
 
 @deprecated_renamed_argument('independent_celestial_slices', None, since='0.6')
 def reproject_interp(input_data, output_projection, shape_out=None, hdu_in=0,
-                     order='bilinear', output_array=None, return_footprint=True):
+                     order='bilinear', output_array=None, return_footprint=True,
+                     independent_celestial_slices=None):
```

Comment:
What happens when someone use these as positional args? The order has been changed compared to 0.5.x. (maybe time to force some kwargs to be kwargs only?)

Reply:
Fixed